<launch>
  <!-- BEGIN ROBOT 0 -->
  <group ns="robot_0">
    <param name="goal_x" value="6.32" />
    <param name="goal_y" value="17.67" />
    <param name="goal_theta" value="0" />
    <node pkg="navigation_multi" type="send_goal" respawn="false" name="send_goal" output="screen" args="0"/>
  </group>
  <!-- END ROBOT 0 -->

  <!-- BEGIN ROBOT 1 -->
  <group ns="robot_1">
    <param name="goal_x" value="10.12" />
    <param name="goal_y" value="12.97" />
    <param name="goal_theta" value="45" />
    <node pkg="navigation_multi" type="send_goal" respawn="false" name="send_goal" output="screen" args="1"/>
  </group>
  <!-- END ROBOT 1 -->
</launch>

  

